devices/generic/items/state_preset_speed_item.json

```diff
@@ -0,0 +1,8 @@
+{
+  "schema": "resourceitem1.schema.json",
+  "id": "state/preset_speed",
```

Comment:
Please re-use `state/effectSpeed` from the Xmas lightstrip (which should probably be `state/effect_speed`).